gli/tags.py

```diff
@@ -101,15 +101,16 @@ def degree_assortativity(nx_g):
     """Compute the degree assortativity coefficient."""
     if not check_direct(nx_g):
         nx_g = nx.Graph(nx_g)
-        return nx.degree_pearson_correlation_coefficient(nx_g)
+        out = nx.degree_pearson_correlation_coefficient(nx_g)
+        return f"{out:6f}"
     else:
         dic = {"in", "out"}
         out = []
         for i in dic:
             for j in dic:
-                out.append(nx.degree_pearson_correlation_coefficient(
-                    nx_g, x=i, y=j))
-        return out
+                out.append(np.round(nx.degree_pearson_correlation_coefficient(
+                    nx_g, x=i, y=j), 6))
+        return f"{out[0], out[1], out[2], out[3]}"
```

Comment:
I feel that returning the numerical value instead of a string is preferred. For example, someone may want to reuse these metric functions at somewhere else. A numerical value output is much more user-friendly.

Reply:
Sure.